pkg/schedule/operator/operator.go

```diff
@@ -149,6 +149,39 @@ func (o *Operator) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + o.String() + `"`), nil
 }
 
+// OpObject is used to return Operator as a json object for API.
+type OpObject struct {
+	Desc        string              `json:"desc"`
+	Brief       string              `json:"brief"`
+	RegionID    uint64              `json:"region_id"`
+	RegionEpoch *metapb.RegionEpoch `json:"region_epoch"`
+	Kind        OpKind              `json:"kind"`
+	Timeout     string              `json:"timeout"`
+	Status      OpStatus            `json:"status"`
+}
+
+// ToJSONObject serializes Operator as JSON object.
+func (o *Operator) ToJSONObject() *OpObject {
+	var status OpStatus
+	if o.CheckSuccess() {
+		status = SUCCESS
```

Comment:
Shall we make it readable instead of printing the number?

Reply:
The method is designed to be used by applications. So I think the number would be easier to be handled.